dvc/progress.py

```diff
@@ -58,12 +62,10 @@ def __init__(
         kwargs  : anything accepted by `tqdm.tqdm()`
         """
         kwargs = kwargs.copy()
-        kwargs.setdefault("unit_scale", True)
         if bytes:
-            bytes_defaults = dict(
-                unit="B", unit_scale=True, unit_divisor=1024, miniters=1
-            )
-            kwargs = merge(bytes_defaults, kwargs)
+            kwargs = merge(self.BYTES_DEFAULTS, kwargs)
```

Comment:
If you are already changing this one:
```suggestion
            kwargs = {**self.BYTES_DEFAULTS, **kwargs}
```